code/modules/jobs/job_types/assistant/gimmick_assistants.dm

```diff
@@ -0,0 +1,178 @@
+// This file contains the louadouts for the assistant gimmicks station trait! When active, gives assistants random stupid gear
+
+/// Parent type of gimmick loadouts for assistants for the functional assistant station traits
+/datum/outfit/job/assistant/gimmick
+	name = "Gimmick Assistant"
+	/// The weight of the outfit to be picked
+	var/outfit_weight = 0
+
+/datum/outfit/job/assistant/gimmick/post_equip(mob/living/carbon/human/equipped, visualsOnly)
+	. = ..()
+
+	// Throw up any hoods
+	var/obj/item/clothing/suit/hooded/hood_suit = locate(/obj/item/clothing/suit/hooded) in equipped
+	if(!hood_suit)
+		return
+
+	var/datum/component/toggle_attached_clothing/component = hood_suit.GetComponent(/datum/component/toggle_attached_clothing)
+	if(!component)
+		return
+
+	component.toggle_deployable()
```

Comment:
hoods are already coded to put themselves on when equipped by an outfit, no?

Reply:
COMSIG_ITEM_EQUIPPED_AS_OUTFIT on /datum/component/toggle_attached_clothing

Reply:
Huh, never crossed my mind it'd already be doing that